src/20240211_AaveV3Ethereum_ChaosLabsRiskParameterUpdatesIncreaseDebtCeilingForSNXAndMKROnV3Ethereum01312024/ChaosLabsRiskParameterUpdatesIncreaseDebtCeilingForSNXAndMKROnV3Ethereum01312024.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Chaos Labs Risk Parameter Updates - Increase Debt Ceiling for SNX and MKR on V3 Ethereum - 01.31.2024"
+author: "Chaos Labs"
+discussions: "https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-increase-debt-ceiling-for-snx-and-mkr-on-v3-ethereum-01-31-2024/16480/1"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xe1a36b7daaf5ab8555510edf53fc75645c7a0ac26b3d47cfe9295b94f96bcf3a"
+---
+
+## Simple Summary
+
+A proposal to increase the debt ceiling of SNX and MKR on V3 Ethereum
+
+## Motivation
+
+SNX
+The current debt ceiling for SNX on V3 Ethereum set at $2.5M, has reached 94% utilization.
+
+Given current market conditions our Isolation Mode Methodology 1 supports increasing the debt ceiling to $4M.
+
+It is important to note that the majority of SNX debt positions are concentrated within two accounts, accounting for over 90% of the total debt, as outlined below. While this concentration does not affect the current recommendation, it is something to continue monitoring and will be considered in future recommendations and cases of significant market changes.
+
+Positions Analysis
+There are currently two major users utilizing SNX as collateral on V3 Ethereum:
+
+1. 0x09d86d566092bec46d449e72087ee788937599d2 1 - borrowing $1.81M in stables (USDC and USDT) against his 1.4M SNX collateral, accounting for nearly 73% of the debt ceiling. The user’s current health score is 1.59.
```

Comment:
Could be good to have links to etherscan on these addresses

Reply:
@sendra Added etherscan links for all wallets